Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/ExposureDetectionConfig.kt

```diff
@@ -3,9 +3,13 @@ package de.rki.coronawarnapp.appconfig
 import com.google.android.gms.nearby.exposurenotification.ExposureConfiguration
 import de.rki.coronawarnapp.appconfig.mapping.ConfigMapper
 import de.rki.coronawarnapp.server.protocols.internal.ExposureDetectionParameters
+import org.joda.time.Duration
 
 interface ExposureDetectionConfig {
 
+    val maxExposureDetectionsPerDay: Int
+    val minTimeBetweenDetections: Duration
+
     val exposureDetectionConfiguration: ExposureConfiguration
```

Comment:
What is this doing here?

Reply:
Direct exposure of the protobuf class as a workaround until specific mapping is available, will check whether we can remove it already. But I think it's used by the ENF client, and there is no specific mapping available yet, and because it will be changed with 1.8.x, not introducing an abstraction here now, saves us some conflicts.